site/ko/beta/guide/distribute_strategy.ipynb

```diff
@@ -0,0 +1,975 @@
+        "id": "94BkvkLInkKd"
+      },
+      "outputs": [],
+      "source": [
+        "with mirrored_strategy.scope():\n",
+        "  dataset = tf.data.Dataset.from_tensors(([1.], [1.])).repeat(1000).batch(\n",
+        "      global_batch_size)\n",
+        "  dist_dataset = mirrored_strategy.experimental_distribute_dataset(dataset)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "grzmTlSvn2j8"
+      },
+      "source": [
+        "그리고 나서는 한 단계의 훈련을 정의합니다. 그래디언트를 계산하기 위해 `tf.GradientTape`를 사용합니다. 이 그래디언트를 적용하여 우리 모델의 변수를 갱신하기 위해서는 옵티마이저를 사용합니다. 분산 훈련을 위하여 이 훈련 작업을 `step_fn` 함수 안에 넣습니다. 그리고 `step_fn`을 앞에서 만든 `dist_dataset`에서 얻은 입력 데이터와 함께 `tf.distrbute.Strategy.experimental_run_v2`에 넣습니다."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "NJxL5YrVniDe"
+      },
+      "outputs": [],
+      "source": [
+        "@tf.function\n",
+        "def train_step(dist_inputs):\n",
+        "  def step_fn(inputs):\n",
+        "    features, labels = inputs\n",
+        "\n",
+        "    with tf.GradientTape() as tape:\n",
+        "      logits = model(features)\n",
+        "      cross_entropy = tf.nn.softmax_cross_entropy_with_logits(\n",
+        "          logits=logits, labels=labels)\n",
+        "      loss = tf.reduce_sum(cross_entropy) * (1.0 / global_batch_size)\n",
+        "\n",
+        "    grads = tape.gradient(loss, model.trainable_variables)\n",
+        "    optimizer.apply_gradients(list(zip(grads, model.trainable_variables)))\n",
+        "    return cross_entropy\n",
+        "\n",
+        "  per_example_losses = mirrored_strategy.experimental_run_v2(\n",
+        "      step_fn, args=(dist_inputs,))\n",
+        "  mean_loss = mirrored_strategy.reduce(\n",
+        "      tf.distribute.ReduceOp.MEAN, per_example_losses, axis=0)\n",
+        "  return mean_loss"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "yRL5u_NLoTvq"
+      },
+      "source": [
+        "위 코드에서 몇 가지 더 짚어볼 점이 있습니다.\n",
+        "\n",
+        "1. 손실(loss)을 계산하기 위하여 `tf.nn.softmax_cross_entropy_with_logits`를 사용하였습니다. 그리고 손실의 합을 전체 배치 크기로 나누는 부분이 중요합니다. 이는 모든 복제된 훈련이 동시에 이루어지고 있고, 각 단계에 훈련이 이루어지는 입력의 수는 전체 배치 크기와 같기 때문입니다. 따라서 손실 값은 각 복제된 작업 내의 배치 크기가 아니라 전체 배치 크기로 나누어야 맞습니다.\n",
+        "2. `tf.distribute.Strategy.experimental_run_v2`에서 반환된 결과를 모으기 위하여 `tf.distribute.Strategy.reduce` API를 사용하였습니다. `tf.distribute.Strategy.experimental_run_v2`는 전략의 각 복제본에서 얻은 결과를 반환합니다. 그리고 이 결과를 사용하는 방법은 여러 가지가 있을 수 있습니다. 종합한 결과를 얻기 위하여 `reduce` 함수를 사용할 수 있습니다. `tf.distribute.Strategy.experimental_local_results` 함수로 각 복제본에서 얻은 결과의 값들 목록을 얻을 수도 있습니다.\n",
+        "3. 분산 전략 범위 안에서 `apply_gradients` 함수가 불리면, 평소와는 동작이 다릅니다. 구체적으로는 동기화된 훈련 도중 각 병렬화된 작업에서 그래디언트를 적용하기 전, 모든 복제본의 그래디언트를 더한 다음, 이 값을 적용하게 됩니다."
```

Comment:
'함수가 불리면' --> '함수가 호출되면'.

'이 값을 적용하게 됩니다' is not necessary. 
If this phrase is necessary, you'd better to avoid '~전, ~다음, ~'.
I suggest '구체적으로는 동기화된 훈련 도중 각 병렬화된 작업에서 그래디언트를 적용하기 전, 모든 복제본의 그래디언트를 더한 다음, 이 값을 적용하게 됩니다' 
-->
'구체적으로는 동기화된 **훈련 중에 병렬화된 각** 작업에서 그래디언트를 적용하기 전에, 모든 복제본의 **그래디언트가 더해집니다**'.